src/vm/arm/stubs.cpp

```diff
@@ -447,10 +447,9 @@ void UpdateGCWriteBarriers(bool postGrow = false)
     BYTE *pbAlteredRange;
     DWORD cbAlteredRange;
     ComputeWriteBarrierRange(&pbAlteredRange, &cbAlteredRange);
-    FlushInstructionCache(GetCurrentProcess(), pbAlteredRange, cbAlteredRange);
```

Comment:
Since this was unconditional, I'm always returning `flushXrestart | ICACHE_FLUSH` from `UpdateGCWriteBarriers` callers.